test/unit/com/unifina/controller/security/ProfileControllerSpec.groovy

```diff
@@ -37,6 +37,7 @@ class ProfileControllerSpec extends Specification {
 			name: "Test User",
 			password:springSecurityService.encodePassword("foobar123!"),
 			enabled: true,
+			lastLogin: new Date(),
```

Comment:
Can remove `lastLogin` if you set the default field value in `SecUser.groovy` (also elsewhere in this file)